setup.py

```diff
@@ -192,7 +192,6 @@ def run(self):
                     cmd = cmd['--parallel'][nprocs]
                 elif not sys.platform.startswith('win'):
                     cmd = cmd['--']['-j'][nprocs]
-            cmd & FG
```

Comment:
Is that why CMake was running twice every time I was building the Python package? Good fix!